### PR TITLE
Automatic update of fbcode/onnx to 602bd622b66bdb4911efed95b74747cc9b179174

### PR DESCRIPTION
**Pull request by houseroad**
Summary:
Previous import was 04a29addfd5b912812addb8dea5f8763fbfaad01

Included changes:
- **[602bd622](https://github.com/onnx/onnx/commit/602bd622)**: Add section about external tensor data to IR.md (#2323) <Jonny Shipton>
- **[165c3f3b](https://github.com/onnx/onnx/commit/165c3f3b)**: Add integer support to Clip (#2532) <Jonny Shipton>
- **[a5fabf87](https://github.com/onnx/onnx/commit/a5fabf87)**: Add operators LessOrEqual and GreaterOrEqual (as functions) (#2606) <Jeremy Cochoy>
- **[f1dcdafc](https://github.com/onnx/onnx/commit/f1dcdafc)**: Fix input document of quantized operators (#2117) <Takeshi Watanabe>
- **[43af9b69](https://github.com/onnx/onnx/commit/43af9b69)**: Add reference impl for sequence ops (#2380) <Bowen Bao>
- **[aa50aa12](https://github.com/onnx/onnx/commit/aa50aa12)**: Print value case of TypeProto more friendly (#2422) <Takeshi Watanabe>
- **[2e67bfc3](https://github.com/onnx/onnx/commit/2e67bfc3)**: Fix issue #2436 (#2447) <daquexian>
- **[d27ffc6b](https://github.com/onnx/onnx/commit/d27ffc6b)**: Add support for integer tensors to Min and Max (#2608) <Jonny Shipton>
- **[5cc668af](https://github.com/onnx/onnx/commit/5cc668af)**: Update IR.md to describe training extension. (#2615) <G. Ramalingam>
- **[8c5bf9d4](https://github.com/onnx/onnx/commit/8c5bf9d4)**: Generate node backend tests for celu operator (#2607) <Jonny Shipton>
- **[7b65287e](https://github.com/onnx/onnx/commit/7b65287e)**: Change dtype of dd_da in gradient test to float32 (#2620) <Shinichiro Hamaji>
- **[e91739f2](https://github.com/onnx/onnx/commit/e91739f2)**: Introduce SoftmaxCrossentropy as a loss function (#2573) <Ksenija Stanojevic>
- **[b008ed3a](https://github.com/onnx/onnx/commit/b008ed3a)**: Support gathernd with batch_dim mode (#2585) <wezuo>
- **[d2fe4f22](https://github.com/onnx/onnx/commit/d2fe4f22)**: Introduce MeanSquaredError as Loss Function (#2570) <Ksenija Stanojevic>
- **[10b812a6](https://github.com/onnx/onnx/commit/10b812a6)**: Add support for default attributes within FunctionExpandHelper (#2588) <Ewa Tusień>
- **[3368834c](https://github.com/onnx/onnx/commit/3368834c)**: adding version update content. (#2609) <Ke Zhang>
- **[8873cb02](https://github.com/onnx/onnx/commit/8873cb02)**: Adding Inverse Op (#2578) <Negin Raoof>
- **[807c62cf](https://github.com/onnx/onnx/commit/807c62cf)**: Training Proposal: Spec Changes and Gradient Operator (#2314) <Wei-Sheng Chin>
- **[9fdae4c6](https://github.com/onnx/onnx/commit/9fdae4c6)**: Copy sizes in some optimizers to remain shape information (#2574) <daquexian>
- **[c978d102](https://github.com/onnx/onnx/commit/c978d102)**: Implement CELU node as a Function (#2575) <Jeremy Cochoy>
- **[c677aef4](https://github.com/onnx/onnx/commit/c677aef4)**: Fix CI build break (#2603) <Changming Sun>
- **[d343755d](https://github.com/onnx/onnx/commit/d343755d)**: Allow function body to rely on other operator sets (#2597) <Ke Zhang>

Test Plan: ci

Differential Revision: D20124936

### CODE REVIEW

**Comment by garymm** [comment]
Obsolete, closing.
